calico_cni/calico_cni.py

```diff
@@ -265,39 +267,57 @@ def _assign_ip(self):
 
         try:
             # Load the response and get the assigned IP address.
+            _log.debug("IPAM plugin result: %s", self.ipam_result)
```

Comment:
_call_ipam_plugin() already logs this out, so I don't think we need to do this here.

Also, `self.ipam_result` will always be `None` at this point, since it hasn't been set yet.